app-admin/lnav/files/lnav-0.10.0-disable-remote-tests.patch

```diff
@@ -0,0 +1,75 @@
+--- lnav-0.10.0/test/test_remote.sh
++++ lnav-0.10.0/test/test_remote.sh
+@@ -76,38 +76,38 @@
+ error: unable to open file: nonexistent-host: -- failed to ssh to host: ...
+ EOF
+
+-run_test ${lnav_test} -d /tmp/lnav.err -n \
+-    localhost:nonexistent-file
++# run_test ${lnav_test} -d /tmp/lnav.err -n \
```

Comment:
I don't want to hold up the PR over this and it's not a big deal, but for future, we can actually just drop lines (no need to comment) because the diff will preserve their contents as - lines anyway.

Also, /tmp is suspicious, we should allow using ${T} from the ebuild environment.